### ISSUE TITLE
Unable to translate custom attributes in admin backend

### ISSUE THREAD

**Issue by cmuench** [magento/magento2#13250]
### Preconditions
A custom attribute is created.
We have different frontend labels for the attribute. Store views with different locale settings exist.
Admin users are speaking different languages and need a localized attribute label.

### Steps to reproduce
Create an custom attribute with different frontend labels.

### Expected result
There are two possible results:

1. A user can see the specific frontend label of the selected store-view. This could be a problem e.g. the user speaks french and the assigned locale of the store view is in another language.

2. The default frontend label can be translated e.g. with the Magento translation functions. This needs a i18n language file or that we can use the inline translation in admin backend.

### Actual result
The default frontend label is used.
Have a look into this method:
`\Magento\Catalog\Ui\DataProvider\Product\Form\Modifier\Eav::setupAttributeMeta`.

```php
$meta = $this->arrayManager->set($configPath, [], [
            'dataType' => $attribute->getFrontendInput(),
            'formElement' => $this->getFormElementsMapValue($attribute->getFrontendInput()),
            'visible' => $attribute->getIsVisible(),
            'required' => $attribute->getIsRequired(),
            'notice' => $attribute->getNote(),
            'default' => (!$this->isProductExists()) ? $attribute->getDefaultValue() : null,
            'label' => $attribute->getDefaultFrontendLabel(),  // <-- This is the line
            'code' => $attribute->getAttributeCode(),
            'source' => $groupCode,
            'scopeLabel' => $this->getScopeLabel($attribute),
            'globalScope' => $this->isScopeGlobal($attribute),
            'sortOrder' => $sortOrder * self::SORT_ORDER_MULTIPLIER,
        ]);
```

At least the translation function should be used:

```php
'label' => __($attribute->getDefaultFrontendLabel()),
```

**Comment by magento-engcom-team** [comment]
@cmuench, thank you for your report.
We've acknowledged the issue and added to our backlog.

**Comment by magento-engcom-team** [comment]
Hi @engcom-backlog-nazar. Thank you for working on this issue. 
Looks like this issue is already verified and confirmed. But if your want to validate it one more time, please, go though the following instruction:
- [ ] 1. Add/Edit `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.

- [ ] 2. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 3. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [ ] 4. If the issue is not relevant or is not reproducible any more, feel free to close it.

**Comment by ghost** [comment]
HI @cmuench thank you for you report, this issue has already fixed in 2.3-develop branch -> https://github.com/magento/magento2/pull/13251 and available on 2.3.1 release.
